platforms/corda-ent/configuration/roles/helm_component/vars/main.yaml

```diff
@@ -10,8 +10,10 @@ helm_templates:
   float: float.tpl
   node_registration: node_registration.tpl
   node: node.tpl
-corda_enterprise_image:
-  cenm-networkmap-1.2: adopblockchaincloud0502.azurecr.io/corda/enterprise-networkmap:1.2-zulu-openjdk8u242
-  cenm-notary-1.2: adopblockchaincloud0502.azurecr.io/corda_image_notary_1.2:latest
-  firewall: adopblockchaincloud0502.azurecr.io/corda_image_firewall_4.4:latest
-  node: adopblockchaincloud0502.azurecr.io/corda_image_ent_4.4:latest
+docker_images:
+  cenm:
+    networkmap-1.2: corda/enterprise-networkmap:1.2-zulu-openjdk8u242
```

Comment:
nmap is also from private repo. Actually all are from private repo

Reply:
@suvajit-sarkar and @sauveergoel Please ensure to remove adopblockchaincloud0502 references in your stories